static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -203,14 +203,30 @@ const bootStandard = (): void => {
     // Set adtest query if url param declares it
     setAdTestCookie();
 
-    // Remove the old ad-free cookie - we can delete this line in a few days
```

Comment:
Maybe put a 'delete after' date in this comment to make it obvious to anyone finding it when it can be deleted?

Reply:
Fair comment. Particularly for this project! That line is months old now, so it's gone.